### PR TITLE
Bug fix/arena score helper

### PR DESCRIPTION
**Pull request by boscohyun**
- Related PR: https://github.com/planetarium/lib9c/pull/622
- Solve a problem when calculating arena score.

|x|win(design)|lose(design)|win(current)|lose(current)|
|-|-----------|------------|------------|-------------|
|x < -500|60|-5|60|-5|
|x < -400|50|-5|**60**|-5|
|x < -300|40|-6|**60**|**-5**|
|x < -200|30|-6|**60**|**-5**|
|x < -100|25|-8|**60**|**-5**|
|x < 0|20|-8|**60**|**-5**|
|x < 100|15|-10|15|-10|
|x < 200|15|-10|**8**|-10|
|x < 300|8|-20|**4**|-20|
|x < 400|4|-25|**2**|-25|
|x < 500|2|-30|**1**|-30|
|x >= 500|1|-30|1|-30|

- **x**: challenger score - defender score
- This PR will be merged into the `development` branch after the `rc-v100075` branch is back-merged.

### CODE REVIEW

File: Lib9c/Model/State/WeeklyArenaState.cs [hunk 1]
```diff
@@ -474,6 +474,31 @@ public int UpdateV3(AvatarState avatarState, ArenaInfo enemyInfo, BattleLog.Resu
...
+            {
+                case BattleLog.Result.Win:
+                    ArenaRecord.Win++;
+                    break;
+                case BattleLog.Result.Lose:
+                    ArenaRecord.Lose++;
+                    break;
+                case BattleLog.Result.TimeOver:
+                    ArenaRecord.Draw++;
+                    return 0;
```
Comment (ipdae) [inline Lib9c/Model/State/WeeklyArenaState.cs]: I think dailyChallengeCount doesn't update this case, is this correct?

File: Lib9c/Action/RankingBattle6.cs [hunk 2]
```diff
@@ -17,6 +17,7 @@
 namespace Nekoyume.Action
 {
     [Serializable]
+    [ActionObsolete(0)]
```
Comment (ipdae) [inline Lib9c/Action/RankingBattle6.cs]: i think we don't need obsolete

File: Lib9c/Action/RankingBattle6.cs [hunk 3]
```diff
@@ -48,6 +49,8 @@ public override IAccountStateDelta Execute(IActionContext context)
                     .SetState(questListAddress, MarkChanged);
             }
 
+            CheckObsolete(0, ctx);
```
Comment (ipdae) [inline Lib9c/Action/RankingBattle6.cs]: i think we don't need obsolete

File: Lib9c/Model/State/WeeklyArenaState.cs (hunk 1 above)
Comment (boscohyun) [reply Lib9c/Model/State/WeeklyArenaState.cs]: Oh..! Thanks to find that!

**Review by ipdae — APPROVED** [review]

**Pull request merged**